project-reports/2023/2023-Q1-Hyperledger-Besu.md

```diff
@@ -0,0 +1,56 @@
+
+Additionally, with Shanghai enabling Ether withdrawals from validators, HLF can begin to use some of these funds to pay for project infrastructure and hopefully some incentives for community engagement down the line alongside CSI and other maintainer groups. 
+
+# Required Information
+All completed for Besu. 
+
+- Have you switched from master to main in all your repos?
+- Have you implemented the [Common Repository Structure](../guidelines/repository-structure.md) in all your repos?
+- Has your project implemented these inclusive language changes listed below to your repo? You can optionally [use the DCI Lint tool](https://github.com/petermetz/gh-action-dci-lint#usage) to make this a recurring action on your repo.
+  - master → main
+  - slave → replicas
+  - blacklist → denylist
+  - whitelist → allowlist
+- Have you added an [Inclusive Language Statement](https://wiki.hyperledger.org/display/TSC/Inclusive+Language+Example) to your project's documentation and/or Wiki pages?
+
+# Questions/Issues for the TSC
+
+Will the TSC deliver a unified CI/CD strategy across HLF projects to avoid ambiguity going forward? 
+
+# Releases
+
+Besu was late in releasing its Quarter 1 CalVer quartlerly (23.1.0), due to huge changes to the codebase and a few bugs with Bonsai specifically. The group released 3 updates in this quarter, 23.1.0, 23.1.1, and 23.1.2. 23.1.2 contains the Bonsai refactor and Shanghai hard-fork support for Ethereum Mainnet. 
+
+# Overall Activity in the Past Quarter
+
+Discord is very active as users look for support with both private and public networks. Consensys has worked with new partners to hand over the bulk of private network support but all maintainers are popping in and out to provide valuable support. Some users still ask questions via issue submissions in Git and the mailing list and these are low in volume compared to Discord. We have seen new contirubitons from some external contributors and an uptick of folks dipping into the #besu-contributors channel to look for insight on the codebase. We hope this will continue as we gain traction with contributors. 
+
+# Current Plans
+
+See the [roadmap]([url](https://wiki.hyperledger.org/display/BESU/Roadmap)). This is primarily maintained by Consensys but is actively looking for feedback and contributions. The project is currently focused around public networks, but hope to continue to onboard contributors to pick up the private network feature development and bug work. 
+
+
+# Maintainer Diversity
+
+No maintainers added, Web3Labs contributors getting up to speed in Q1 and might soon hit the threshold of commits for Maintainer status. 
+
+# Contributor Diversity
+
+Too many to user orgs to list - not many converting to contributor status. Maintainers are actively working with HLF leadership to change this. 
```

Comment:
```suggestion
Too many user orgs to list - not many converting to contributor status. Maintainers are actively working with HLF leadership to change this. 
```

Reply:
it might be good to add a link to the [insights report for 1/1/2023-3/31/2023](https://tinyurl.com/2bsm6dt5) so that people can take a look at the contributions for the quarter.